src/renderer/viz/expressions/color/hsv.js

```diff
@@ -119,6 +119,15 @@ function genHSV (name, alpha) {
             return hsvToRgb(h, s, v);
         }
 
+        getLegendData () {
```

Comment:
@elenatorro This looks as a good candidate for a shared helper function, as the implementation is the same for several color types.